extra_tests/regression_fits/polarized_evol.yml

```
@@ -0,0 +1,82 @@
+############################################################
+# Configuration file for N3FIT regression Test
+############################################################
+description: N3FIT regression test for polarised fits
+
+############################################################
+dataset_inputs:
+- {dataset: E143_NC_NOTFIXED_EP_G1, frac: 0.60, cfac: [NRM]}
+- {dataset: E143_NC_NOTFIXED_ED_G1, frac: 0.60, cfac: [NRM]}
+- {dataset: E154_NC_9GEV_EN_G1, frac: 0.60, cfac: [NRM]}
+
+############################################################
+datacuts:
+  t0pdfset: NNPDFpol10_100   # PDF set to generate t0 covmat
+  q2min: 1.00                # Q2 minimum
+  w2min: 4.00                # W2 minimum
+
+############################################################
+# Define the unpolarized PDF set to be used as BC for POS
+positivity_bound:
+  unpolarized_bc: NNPDF40_nnlo_pch_as_01180
+  n_std: 1.00 # Standard Deviation to be added as Error
+
+############################################################
+theory:
+  theoryid: 822
+
+############################################################
+trvlseed: 14613416
+nnseed: 1865683875
+mcseed: 746898422
+genrep: true
+
+save: "weights_polevol.h5"
```

```suggestion
save: "weights_polevol.weights.h5"
```

Are you sure you want to `save` here and not load a previously saved state? (note that there are already other two cards that test the save)

---

I think this is already outdated. In the recent commits, this has been changed to load.